sonar-application/src/test/java/org/sonar/application/AppTest.java

```diff
@@ -63,14 +63,50 @@ public void start_all_processes_by_default() throws Exception {
     verify(monitor).start(argument.capture());
 
     assertThat(argument.getValue()).extracting("processId").containsExactly(ProcessId.ELASTICSEARCH, ProcessId.WEB_SERVER, ProcessId.COMPUTE_ENGINE);
+
+    app.stopAsync();
+  }
+
+  @Test
```

Comment:
@simonbrandhof I think this is missing a test that makes sure that properties `sonar.cluster.xxx.disabled=true` are ignored when `sonar.cluster.enabled=false` or is not set